src/promptflow-evals/promptflow/evals/_constants.py

```diff
@@ -14,11 +14,13 @@ class EvaluationMetrics:
 
 
 class Prefixes:
-    _INPUTS = 'inputs.'
-    _OUTPUTS = 'outputs.'
-    _TGT_OUTPUTS = '__outputs.'
+    _INPUTS = "inputs."
+    _OUTPUTS = "outputs."
+    _TGT_OUTPUTS = "__outputs."

```

Comment:
btw, what's TGT?

Reply:
I believe TGT is the abbreviation for Target